frontend/src/app/components/objective-detail/objective-detail.component.ts

```diff
@@ -48,6 +54,15 @@ export class ObjectiveDetailComponent implements OnInit {
       .subscribe((objective) => this.objective$.next(objective));
   }
 
+  loadComponent(id: number): void {
+    this.completedService
+      .getCompleted(id)
```

Comment:
Don't load the completed all the time.
You get an error if the objective isnt completet